src/service/angular-sails.js

```diff
@@ -1,74 +1,168 @@
-/*jslint sloppy:true*/
-/*global angular, io */
-angular.module('ngSails').provider('$sails', function () {
-    var provider = this,
-        httpVerbs = ['get', 'post', 'put', 'delete'],
-        eventNames = ['on', 'once'];
+(function(angular, io) {
+  'use strict';
+  io.sails.autoConnect = false;
+  window.io.sails.useCORSRouteToGetCookie = false;
```

Comment:
window is not needed here are the `io` in scope is from the window.  Also assumes newer version of sails.io